src/roiextractors/extraction_tools.py

```diff
@@ -31,6 +31,7 @@
 DtypeType = DTypeLike
 IntType = Union[int, np.integer]
 FloatType = float
+NoneType = type(None)
```

Comment:
Is this used? Is it needed? I think generally `None` is used in typehinting to express the `None` type